jvm/adapter/src/main/java/com/oracle/truffle/js/lang/JavaScriptLanguageHack.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2011-2024 GatlingCorp (https://gatling.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.oracle.truffle.js.lang;
+
+import static io.gatling.internal.asm.Opcodes.*;
+
+import io.gatling.internal.asm.*;
+import java.io.IOException;
+import java.lang.invoke.MethodHandles;
+
+public class JavaScriptLanguageHack {
```

Comment:
I might rename this. 😅 It does need to be in the package `com.oracle.truffle.js.lang`, because calling `MethodHandles.lookup().defineClass()` from a different package from the class being defined is not allowed (it throws an exception saying so).